pycbc/waveform/waveform.py

```diff
@@ -514,7 +514,10 @@ def get_sgburst_waveform(template=None, **kwargs):
 _inspiral_fd_filters = {}
 _cuda_fd_filters = {}
 
-from spa_tmplt import spa_tmplt
+from spa_tmplt import spa_tmplt, spa_tmplt_norm, spa_tmplt_end, \
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->